### DISCUSSION TITLE
Improve Recommendation engine performance

### DISCUSSION THREAD

**Discussion by BurakKocak99** [LivingStories-DevTeam/Living-Stories-App#144] [General]
The performance of the recommendation engine should be improved for usability purposes.

**Comment by JohnsL-U** [comment]
I believe the issue to be front end related rather than recommendation engine itself. Feel free to test the beta version of the recommendation engine by using the following template and entering an user id number

`http://35.246.216.172:5002/recommendations?user_id=`

**Comment by JohnsL-U** [comment]
I realized that our live server has very limited number of user interactions, users, stories. As a result, it is difficult to gauge the limitations of recommendation engines performance. I suggest deleting meaningless users, increasing the user interactions significantly for the remaining users and then evaluating the recommendation engine's performance again.
